libs/@local/graph/postgres-store/src/store/postgres/knowledge/entity/mod.rs

```diff
@@ -1172,11 +1060,14 @@ where
             .await
             .change_context(InsertionError)?;
 
-        let validator_provider = StoreProvider {
-            store: &transaction,
-            cache: store_cache,
-            authorization: Some((actor_id, Consistency::FullyConsistent)),
-        };
+        // let validator_provider = StoreProvider {
```

Comment:
should this stay commented out?

Reply:
Removed in https://github.com/hashintel/hash/pull/7352/commits/12733a765309583bd55fa59aef36e37d0c525d1b. This was the reason why I removed the `mut` in https://github.com/hashintel/hash/pull/7352#discussion_r2132034191 so we can re-use the already existing provider.